docs/RELEASE-NOTES.rst

```diff
@@ -4,6 +4,22 @@ Release Notes for BIG-IP Controller for Kubernetes
 |release|
 ---------
 
+Added Functionality
+```````````````````
+
+* Watch all nodes by default; watch nodes with a user-specified label.
+* Custom SSL Profiles can be loaded from Kubernetes Secrets via Ingress TLS.
+* Creation of BIG-IP objects from OpenShift Route resources.
+  - This includes unsecured, Edge, Passthrough, and Re-encrypt Routes
+* Can properly configure http redirect rules on v11.6.1 BIG-IP systems
```

Comment:
Is this a bugfix?

Reply:
Yes, should I include a Bug Fixes section?